.env.sample

```diff
@@ -4,18 +4,31 @@ CONFIG_SERVICE_URI=https://safe-config.staging.gnosisdev.com
 # Production
```

Comment:
I wouldn't make our staging urls public

Reply:
I agree. This was already there but it makes sense to remove it.